rero_ils/modules/documents/jsonschemas/documents/document_scale_cartographic-v0.0.1.json

```diff
@@ -0,0 +1,180 @@
+    "type": "array",
+    "minItems": 1,
+    "items": {
+      "title": "Scale",
+      "type": "object",
+      "propertiesOrder": [
+        "type",
+        "label",
+        "ratio_linear_horizontal",
+        "ratio_linear_vertical"
+      ],
+      "required": [
+        "type",
+        "label",
+        "ratio_linear_horizontal"
+      ],
+      "properties": {
+        "type": {
+          "title": "Type",
+          "type": "string",
+          "default": "Linear scale",
+          "enum": [
+            "Linear scale",
+            "Angular scale",
+            "Other"
+          ],
+          "form": {
+            "Type": "selectWithSort",
+            "options": [
+              {
+                "label": "Linear scale",
+                "value": "Linear scale"
+              },
+              {
+                "label": "Angular scale",
+                "value": "Angular scale"
+              },
+              {
+                "label": "Other",
+                "value": "Other"
+              }
+            ]
+          }
+        },
+        "ratio_linear_horizontal": {
+          "title": "Horizontal scale",
+          "description": "Enter only the denominator.",
+          "type": "string",
+          "minLength": 1,
+          "form": {
+            "placeholder": "Example: 25000"
+          }
+        },
+        "ratio_linear_vertical": {
+          "title": "Vertical scale",
+          "description": "Enter only the denominator.",
+          "type": "string",
+          "minLength": 1,
+          "form": {
+            "placeholder": "Example: 250"
+          }
+        },
+        "label": {
+          "title": "Label",
+          "description": "Scale designation (MARC 255$a)",
+          "type": "string",
+          "minLength": 1,
+          "form": {
+            "placeholder": "Example: Scale 1:25 000"
+          }
+        }
+      }
+    },
+    "form": {
+      "hide": true
+    }
+  },
+  "cartographicAttributes": {
+    "title": "Cartographic datas",
+    "type": "array",
+    "minItems": 1,
+    "items": {
+      "title": "Cartographic data",
+      "type": "object",
+      "propertiesOrder": [
+        "coordinates",
+        "projection",
+        "equinox"
+      ],
+      "required": [
+        "coordinates"
+      ],
+      "properties": {
+        "coordinates": {
+          "title": "Coordinates",
+          "type": "object",
+          "propertiesOrder": [
+            "label",
+            "longitude",
+            "latitude",
+            "declination",
+            "rightAscension"
+          ],
+          "required": [
+            "label",
+            "longitude",
+            "latitude"
+          ],
+          "properties": {
+            "label": {
+              "title": "Label",
+              "type": "string",
+              "minLength": 1
+            },
+            "longitude": {
```

Comment:
For **longitude, latitude, declination, right_ascension**
-> we specified regex for the type. See spec column N.